test/run-stp-tests.sh

```diff
@@ -443,14 +443,12 @@ EOF
 }
 
 ln -s $root/test/test.convs $BASE/share/mime
-ln -s $root/test/test.types $BASE/share/mime
```

Comment:
We need a MIME types file for the test...

Reply:
Only for pdftourf.